grafana-plugin/src/containers/Rotations/Rotations.styles.ts

```diff
@@ -0,0 +1,93 @@
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getRotationsStyles = (theme: GrafanaTheme2) => {
+  return {
+    root: css`
+      background: 1px solid ${theme.colors.background.secondary};
+      border: ${theme.colors.border.weak};
+      display: flex;
+      flex-direction: column;
+      border-radius: ${theme.shape.radius.default};
```

Comment:
With grafana 9.4 themes, `shape.radius` is not defined.

Could you return to fixed value, thanks.